desk/sur/groups.hoon

```diff
@@ -138,19 +136,29 @@
       zone-ord=(list zone)
       =bloc
       =channels:channel
+      active=(set nest)
       imported=(set nest)
       =cordon
       secret=?
       meta=data:meta
       =flagged-content
   ==
-::
-+$  group-ui  [group saga=(unit saga:e)]
++$  group-ui  [group init=? count=@ud]
 ::  $cabal: metadata representing a $sect or role
 ::
+::TODO rename cabal -> role-meta?
```

Comment:
perhaps cabal -> role and sect -> role-id

Reply:
Good suggestion! Do note that I haven't planned to do these name changes just yet. I will soon do another round on the %groups, this time to survey everything that's in the agent and document endpoints. Name changes could be done at that time.